media-libs/Field3D/Field3D-1.7.3_p20230509.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2023 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake flag-o-matic
+
+DESCRIPTION="A library for storing voxel data"
+HOMEPAGE="http://opensource.imageworks.com/?p=field3d"
+SRC_COMMIT="b0ff0cd67893a4cbfa322676eb3eef10100d904d"
+SRC_URI="https://github.com/imageworks/Field3D/archive/${SRC_COMMIT}.tar.gz -> ${P}.tar.gz"
```

Comment:
am I missing the point here? the only commit between 1.7.3 and this is updating the README?

We should also fix https://bugs.gentoo.org/912312.

Reply:
This is the last commit before archiving the repo. So might as well use that.
Open bugs are fixed.